portfolio/src/main/webapp/index.html

```diff
@@ -25,84 +26,177 @@
+              <select id="new-task-priority" class="form-control" name="new-task-priority">
+                <option value="5">5 (High)</option>
+                <option value="4">4</option>
+                <option value="3">3</option>
+                <option value="2">2</option>
+                <option value="1">1 (Low)</option>
+              </select>
+            </div>
+          </div>
+          <br />
+          <button type="button" class="btn btn-primary" onclick="createNewTask()">Add this task</button>
+        </form>
 
-    <div id="new-event-wrapper">
-      <h2>Add any pre-scheduled events here</h2>
-      <div id="new-event-form">
-        <label for="new-event-name">Event name:</label>
-        <input type="text" id="new-event-name" name="new-event-name" value="New Event" /><br />
-        <span>
-          <label for="new-event-start-time">Start time:</label>
-          <input type="time" id="new-event-start-time" name="new-event-end-time" value="09:00" />
-        </span>
-        <br />
-        <span>
-          <label for="new-event-end-time">End time:</label>
-          <input type="time" id="new-event-end-time" name="new-event-end-time" value="20:00" />
-        </span>
         <br />
-        <button type="button" class="btn btn-primary" onclick="createNewCalendarEvent()">Add this task</button>
+        <h2>Tasks added:</h2>
+        <div class="scrollable" id="new-task-list-wrapper">
+          <div id="new-task-list"></div>
+        </div>
       </div>
-    </div>
 
-    <div id="new-event-list-wrapper">
-      <h2>Events added:</h2>
-      <h3 id="event-end-time-warning">End time must be later than start time.</h3>
-      <div id="new-event-list"></div>
-    </div>
+      <div id="new-event-wrapper" class="col-md-12 col-lg-6 col-xl-3">
+        <h2>Add any pre-scheduled events here</h2>
+        <h4>We will not schedule any tasks during these time slots.</h4>
+        <form id="new-event-form">
+          <div class="form-group">
+            <label for="new-event-name">Event name:</label>
+            <input
+              type="text"
+              class="form-control"
+              aria-describedby="new-event-name"
+              id="new-event-name"
+              name="new-event-name"
+              value="New Event"
+            />
+          </div>
+
+          <div class="row">
+            <div class="col">
+              <div class="form-group">
+                <label for="new-event-start-time">Start time:</label>
+                <input
+                  type="time"
+                  class="form-control"
+                  aria-describedby="new-event-start-time"
+                  id="new-event-start-time"
+                  name="new-event-end-time"
+                  value="09:00"
+                />
+                <small id="new-event-time-help-text" class="form-text text-muted"
+                  >The times you enter will be in your current timezone.</small
```

Comment:
Similarly the format here seems off. Can you run the HTML linter?

Reply:
Done.